packages/manager/src/components/TagCell/TagDrawer.tsx

```diff
@@ -33,48 +22,11 @@ export interface TagDrawerProps {
 export type CombinedProps = Props;
 
 export const TagDrawer: React.FC<Props> = (props) => {
-  const { addTag, entityLabel, deleteTag, onClose, open, tags } = props;
-  // @todo the new tag component should have a loading state for when a tag is being deleted
-  // const [loadingTag, setLoadingTag] = React.useState<string>('');
-  const [tagError, setTagError] = React.useState<string | undefined>();
-
-  const classes = useStyles();
-
-  React.useEffect(() => {
-    if (open) {
-      setTagError(undefined);
-    }
-  }, [open]);
-
-  const _addTag = (tag: string) => {
-    setTagError(undefined);
-    addTag(tag).catch((e) =>
-      setTagError(getAPIErrorOrDefault(e, 'Error adding tag.')[0].reason)
-    );
-  };
-
-  const _deleteTag = (tag: string) => {
-    setTagError(undefined);
-    // setLoadingTag(tag);
-    deleteTag(tag).catch((e) =>
-      setTagError(getAPIErrorOrDefault(e, 'Error deleting tag.')[0].reason)
-    );
-  };
+  const { updateTags, entityLabel, onClose, open, tags } = props;
 
   return (
     <Drawer open={open} title={`Tags (${entityLabel})`} onClose={onClose}>
-      {tagError && <Notice error text={tagError} />}
-      {tags.map((thisTag) => (
-        <Tag
-          key={`tag-item-${thisTag}`}
-          colorVariant="lightBlue"
-          label={thisTag}
-          onDelete={() => _deleteTag(thisTag)}
-        />
-      ))}
-      <div className={classes.addTag}>
-        <AddTag tags={tags} addTag={_addTag} label={'Add a tag'} />
-      </div>
+      <TagsPanel tags={tags} updateTags={updateTags}></TagsPanel>
```

Comment:
TagsPanel is a newer component that offers goodies like progress indication

Reply:
since there are no child components:

```suggestion
<TagsPanel tags={tags} updateTags={updateTags} />
```